logging/log_formatters.go

```diff
@@ -185,3 +216,24 @@ func nanosecondsToMillisecondsString(nanos int64) string {
 func unixTimeWithMilliseconds(e *logrus.Entry) string {
 	return TimeToString(e.Time)
 }
+
+func formatEntry(e *logrus.Entry, formatter log.Formatter, hooks []FormatterHook) ([]byte, error) {
```

Comment:
cool, one readable simple function which provides so many ways to inject some extra logic)

Reply:
@ilammy 's code)))